docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,293 @@
+== Step 3. Add Configuration for the Production Cluster
+Now add the TLS certificate files and a new configuration file for your production cluster.
+
+. Save the TLS certificate files for the production cluster to a separate directory on your local machine. For example: `$HOME/.local/share/*prod*`.
+. Return to the `$HOME/.local/share/hz-cli` directory, make a copy of the `config.yaml` file and save it to the same directory. Give the new file a unique name. For example, `config_prod.yaml`.
+. Open the new configuration file and update the following parameters with values for your production cluster.
++
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/$PATH/ca.pem"
+  certpath: "/$PATH/cert.pem"
+  keypath: "/$PATH/key.pem"
+  keypassword: "$PROD_KEYSTORE_PASSWORD"
+hazelcast:
+  cluster:
+    name: "$PROD_CLUSTER_NAME"
+    cloud:
+      token: "$PROD_DISCOVERY_TOKEN"
+      enabled: true
+----
+. Replace the following placeholders:
+
+** `/$PATH/`: The absolute path to the TLS certificate files on your local machine.
+** `$PROD_KEYSTORE_PASSWORD`: The keystore and truststore password for the production cluster.
+** `$PROD_CLUSTER_NAME`: The short name for the production cluster. For example: `pr-3145`.
+** `$PROD_DISCOVERY_TOKEN`: The discovery token for the production cluster.
+. Save your changes.
+
+[[step-4-cluster-switch]]
+== Step 4. Switch Between Clusters
+Having separate local configurations for your development and production clusters allows you to switch between them without leaving the command line. Make sure both clusters are running before you start.
+
+. From the command line, change into the `$HOME/.local/share/hz-cli` directory.
+. Execute the following command to use the default configuration to connect to your development cluster:
++
+```bash
+hzc
+```
++
+As before, the Hazelcast CLC connects to your development cluster.
+. Type `exit` to return to `$HOME/.local/share/hz-cli`.
+. Execute the following command to use the new configuration file for your production cluster.
++
+```bash
+hzc -c <config_prod>.yaml
+```
++
+The Hazelcast CLC connects to your production cluster.
+
+[[step-5-write-data]]
+== Step 5. Write Data to a Map
+Now that you've connected to both your clusters, try using the Hazelcast CLC to write data to a map on your development cluster. A quick and easy way to do this is to use a JSON file.
+
+. Copy the following JSON to a file in the `$HOME/.local/share/hz-cli` directory.
++
+.currencycodes.json
+```json
+{
+  "1": {
+        "Code": "CAD",
+        "Currency": "Canadian Dollar"
+      },
+  "2": {
+        "Code": "INR",
+        "Currency": "Indian Rupee"
+      },
+  "3": {
+        "Code": "MXN",
+        "Currency": "Mexican Peso"
+      },
+  "4": {
+        "Code": "GBP",
+        "Currency": "Pounds Sterling"
+      },
+  "5": {
+        "Code": "TRY",
+        "Currency": "Turkish Lira"
+      },
+  "6": {
+        "Code": "USD",
+        "Currency": "United States Dollar"
+      },
```

Comment:
```suggestion
      }
```

Otherwise, CLC throws an error:

```
Error: given json map entry file is in invalid format
Details: invalid character '}' looking for beginning of object key string
```

Not sure if this is expected @yuce @utku-caglayan

Reply:
Thanks for letting us now. Your suggestion is the one that complies with json syntax @JakeSCahill. There shouldn't be a semicolon after last element of a json array. Shortly yes, this is an expected error

Reply:
Updated as requested.